github/table_github_code_owner.go

```diff
@@ -0,0 +1,157 @@
+package github
+
+import (
+	"context"
+	"github.com/google/go-github/v45/github"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+)
+
+//// TABLE DEFINITION
+
+func gitHubCodeOwnerColumns() []*plugin.Column {
+	return []*plugin.Column{
```

Comment:
In the columns, can you please add a `line` column that's the line number the entry is found on? Then we can use this column directly instead of needing the `row_number()` function as well in the example.

Reply:
I add this column, but it leads to a non consecutive sequence if we point on the file line number (because we aggregate comments on a rule line).

Are you agree with that ?

Reply:
Yes, that's ok, even if the values of the `line` column across rows become non-consecutive, that's ok. I believe it's still useful so you can tell which rules are in which position in the file, which affects which rules take precedence.

Reply:
Good ! So it's done @cbruno10 :-)